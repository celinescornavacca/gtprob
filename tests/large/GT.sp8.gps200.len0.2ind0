((((((C86,(C97,(C108,C109))),((C171,(C194,C195)),C123)),((((C52,C53),C47),((C173,C174),((C79,(C177,C178)),C29))),((((((((C27,C28),(C188,C189)),(C87,C88)),(C112,C113)),((C21,((C99,C100),C36)),((((((C155,(C184,C185)),(C15,C16)),(C20,(C89,C90))),C2),((C106,C107),(C17,(C58,C59)))),((((((C182,C183),C172),C98),(C158,C159)),(C163,C164)),(C150,C151))))),(C1,(C5,(((C104,C105),(C119,C120)),(((C63,C64),(((C133,C134),(C157,((C167,C168),C162))),(C95,(C128,C129)))),((((C96,((C142,C143),C114)),C72),(C73,C74)),((C137,C138),C62))))))),(((((((C54,C55),(C84,C85)),(((C198,C199),C156),C48)),(((C39,C40),(((C152,C153),C141),(C68,C69))),((((C192,C193),C154),(C144,C145)),(C42,(C124,C125))))),((C196,C197),C18)),((C24,C25),(C34,((C180,C181),(C75,C76))))),(((((C37,C38),((C165,C166),C46)),C22),C6),(C4,((C60,(C91,C92)),(C10,C11)))))),(((C8,(C30,(C33,(((C148,C149),((C126,C127),C115)),C45)))),((C139,C140),(C13,C14))),((C0,(C57,(C77,C78))),(((((C130,C131),((C135,C136),(((C80,C81),C56),C51))),((((C32,((C186,C187),(C175,C176))),(C19,((C146,C147),(C61,(C70,C71))))),C7),(((C93,C94),(((C110,C111),C23),C12)),((C41,(C65,C66)),C9)))),C3),((((((C169,C170),C118),((C190,C191),C179)),((C49,C50),(C43,C44))),(C82,C83)),(((C35,(((C116,C117),((C160,C161),C132)),((C102,C103),(C67,(C101,(C121,C122)))))),C31),C26)))))))),(((((((G93,(G155,G156)),((G190,G191),G6)),((((((G90,G91),G83),((G178,G179),G97)),(G166,G167)),((G109,G110),((G55,(((G126,(G136,G137)),G94),G68)),(G150,(G151,G152))))),(((((G170,G171),G38),(G157,G158)),((G21,(G24,G25)),((G16,G17),((((G177,(G182,G183)),G118),(G99,G100)),(G29,G30))))),((G164,G165),((G27,G28),((G63,G64),G49)))))),((G101,G102),((((G113,(G119,G120)),((G162,G163),(((G22,(((G33,G34),((G196,G197),G46)),G26)),(((G88,G89),((G78,(G121,G122)),((G87,((G172,G173),G98)),((G44,G45),(((((G144,G145),G92),G59),(G142,G143)),G35))))),G8)),((((G84,G85),(G95,G96)),((G180,G181),(G146,G147))),(G7,(G159,G160)))))),((G31,G32),((G39,((G169,((G186,G187),G176)),G161)),(G66,G67)))),(((((G138,G139),G125),G40),G15),(G77,(G124,((G192,G193),G133))))))),(((G69,G70),((((((G131,G132),G14),((G148,G149),((G41,G42),(G61,G62)))),((G2,(((G103,G104),G86),((G47,G48),G18))),((G9,(G11,(G81,G82))),(((G114,G115),(G129,G130)),(G194,G195))))),G1),(((G116,G117),G56),(((((G111,G112),G60),G43),G19),G3)))),(((((G36,G37),((((G65,(G71,(G174,G175))),G50),(G188,G189)),(((G153,G154),(G51,G52)),(((((G75,G76),G74),G20),((G107,(G123,(G140,G141))),((G105,G106),(G108,(G127,G128))))),G10)))),((G134,G135),G5)),(((G12,G13),G4),((((G72,G73),(G79,G80)),(((G184,G185),G168),G23)),(G53,G54)))),(G0,((G198,G199),(G57,G58)))))),((((((((H130,H131),((H50,H51),(H41,((H74,H75),(H85,(H95,(((H123,H124),H109),(H182,H183)))))))),(((H101,H102),H70),(H67,(H81,H82)))),(((H178,H179),H175),(H159,(H184,H185)))),((((H176,H177),H73),H64),((H53,((H151,H152),H60)),(H21,H22)))),(((H112,(H180,H181)),(H27,((H83,H84),(((H149,H150),H108),(H188,H189))))),(H19,((H24,(H135,(H170,H171))),(H138,H139))))),((H2,((H56,H57),(H120,H121))),((((H33,(((H132,H133),H36),H34)),(H155,H156)),H1),((((H52,((H190,H191),H103)),(H76,H77)),(H68,H69)),(H20,((H198,H199),H93)))))),((H3,H4),((H0,((((((H140,(H145,(H162,H163))),((H11,(((H42,(H43,((H110,H111),H63))),((((H153,H154),H55),(H61,H62)),(H35,((H168,H169),H38)))),(H15,H16))),H9)),(H79,H80)),(((H100,(H122,((H148,(H194,H195)),H127))),(H46,(H143,H144))),H10)),((H89,H90),(H166,H167))),((H5,((((H14,(H186,H187)),((H31,H32),(((((H104,H105),H91),(((H196,H197),H172),H94)),H78),((H97,(H114,H115)),(H106,H107))))),(((((H118,H119),H48),H47),(H164,H165)),(((H128,H129),H28),((H192,H193),H44)))),H6)),(((H141,H142),H45),((H71,H72),(H37,(H39,(H49,(H136,H137))))))))),((((((H157,H158),H30),(((H54,(H113,(H160,H161))),(H12,H13)),((H40,(H125,H126)),H29))),(((H98,H99),H96),((((H86,(H87,H88)),H66),H59),(H58,(H134,(H173,H174)))))),((H8,(H17,H18)),((((H116,H117),H92),H23),(H7,((H146,H147),H65))))),(H25,H26)))))),(((F171,F172),F1),(((((F4,(((((F117,F118),(F140,((F164,(F177,F178)),(F173,F174)))),(F182,F183)),(F51,F52)),(((F64,F65),F47),(F156,F157)))),(F42,(F141,F142))),(((F5,(F109,F110)),(F45,((F133,F134),((F123,F124),F108)))),(((F8,((((((F89,F90),F82),F60),((((((F169,F170),F119),F50),F46),(((((F194,F195),F155),F66),(F83,(F186,F187))),F35)),F18)),F15),(((F61,F62),(F86,(F103,F104))),F22))),F7),(((F111,F112),F41),(F27,(((F85,(F137,F138)),F49),F30)))))),(((F54,F55),F21),F20)),((F0,((((F198,F199),F28),F6),((((F120,F121),(F71,F72)),F29),(F79,(F80,F81))))),((((((((F39,F40),((F36,((F162,F163),F68)),F31)),F2),((F179,F180),F53)),(((F190,F191),F78),(F167,F168))),(((F91,F92),((F73,F74),F48)),(F3,((F95,F96),((F160,F161),F84))))),(((((F87,F88),F32),F23),(((((F150,(F151,F152)),F97),((F165,F166),(F196,F197))),F26),F13)),((F158,F159),F24))),(((((((F63,(F69,F70)),(F192,F193)),(F153,F154)),((((((F125,F126),(((F101,F102),F44),(F129,F130))),((F93,F94),(F33,F34))),(((F175,F176),F145),F139)),(((F143,F144),F77),(F135,F136))),(((F16,(F19,(F43,(F98,(F127,F128))))),((F188,F189),F107)),F10))),(((F56,F57),(F37,F38)),(((F75,F76),((F58,F59),(F122,(F146,F147)))),F17))),((F148,F149),F25)),(((F11,F12),F9),((((F113,F114),(F67,((F131,F132),F106))),F14),((F181,(F184,F185)),((F99,F100),((F115,F116),F105)))))))))))),((((B125,B126),(B196,B197)),((((B13,((((B165,B166),B101),((B123,B124),((B136,B137),B105))),(B40,B41))),((B59,B60),B12)),(B20,(B96,B97))),((((((B130,B131),B64),B35),(B33,((B51,(B109,(B171,(B176,B177)))),B50))),((B94,B95),((B133,B134),((B154,B155),(B140,(B148,B149)))))),(((((B132,(B152,B153)),(B118,B119)),(B52,B53)),B36),((((B184,B185),(B23,(B54,(B112,B113)))),B6),(B158,B159)))))),((((B69,B70),(((B163,B164),(B145,B146)),(((B84,B85),(B48,B49)),(B26,B27)))),((((B169,B170),((B7,((B44,B45),((((B194,B195),(B147,(B180,B181))),(B188,B189)),B24))),(((B86,B87),(B150,B151)),(((B107,B108),B68),(B83,(B99,B100)))))),((B3,B4),(B19,(B61,(B71,B72))))),((B1,B2),((((B28,(B129,(B135,(B156,B157)))),((B15,(B186,B187)),(((B82,(B110,B111)),(B78,(B102,(B178,B179)))),B14))),(B103,B104)),((((B160,(B161,B162)),B106),B10),((B34,(B55,(B73,(B122,(B190,B191))))),B32)))))),((((B0,(B8,(B17,(B21,((B80,B81),(B143,B144)))))),((B11,((B29,(((B74,B75),B62),B39)),B25)),(B16,(B37,B38)))),(((((B172,B173),((((B182,B183),(B174,B175)),B57),(B116,B117))),B30),(((B127,B128),(B167,B168)),B56)),((B138,B139),(B141,B142)))),(((B5,(B9,(((B79,(B98,(B198,B199))),B18),(B120,B121)))),((((B92,B93),(B42,B43)),(B65,B66)),(((B67,(B76,B77)),B63),(B114,B115)))),(((B22,(B192,B193)),(B58,((B90,B91),(B88,B89)))),(B31,(B46,B47)))))))),((((((((A29,(A186,A187)),(A9,(((A113,(A132,(A188,A189))),A102),A66))),(((((A130,(A140,A141)),A98),A72),(((A174,A175),A165),(A194,A195))),((A31,A32),(A82,A83)))),((A2,A3),((((A68,A69),A61),((A123,((A190,A191),A152)),A30)),((((A192,A193),(A146,A147)),(A176,A177)),(A52,(A91,A92)))))),(((((A70,A71),(((A115,A116),((A178,A179),A119)),A67)),(A76,(A108,A109))),((A16,(((A180,A181),A101),A73)),(((((A163,A164),A26),((A50,A51),A33)),(A136,A137)),A10))),((A5,(((((A90,(A148,A149)),A24),(A56,A57)),(A198,A199)),((A77,(A84,(A96,A97))),A64))),((A49,(((A106,(A117,A118)),(A78,A79)),(A85,((A133,A134),A95)))),(A22,(A34,A35)))))),(((A55,(A80,A81)),(A17,((A74,A75),(A99,A100)))),(((A4,(((A93,A94),(A126,A127)),((A111,A112),(A37,A38)))),A0),((((((A15,(A39,A40)),((A65,(A182,A183)),(A23,(A87,((A110,(A121,(A124,A125))),(A168,A169)))))),(A19,((A172,A173),A131))),(((A11,A12),(((A170,A171),A13),(((A184,A185),A114),(A43,A44)))),A6)),(A62,A63)),(((A120,(A138,A139)),A25),(A20,A21)))))),((((A45,A46),(A27,A28)),((A36,(A88,A89)),(A166,A167))),((A14,((A47,A48),(A58,A59))),((((((A157,A158),((((A161,A162),(A196,A197)),((A153,A154),A135)),(A53,A54))),(A7,(((A155,A156),(A159,A160)),A86))),((((A144,A145),A105),(A41,A42)),(((A150,A151),A122),A107))),(((A142,A143),A18),(A8,(A60,((A103,A104),(A128,A129)))))),A1)))),(((((((((((((D182,D183),((((D146,(D174,D175)),D115),((D109,D110),D101)),D40)),D23),((D130,D131),D102)),(D14,D15)),((D24,D25),((D198,D199),D173))),((((D83,((D136,D137),D90)),D66),(D150,D151)),((D88,D89),((D45,D46),(D126,D127))))),((((D113,D114),(D19,((D26,D27),D20))),((D17,D18),(((D167,D168),D96),(D116,D117)))),(D70,((D147,(D188,D189)),D71)))),(((((D42,D43),(D86,D87)),D6),((D36,D37),(D38,(D80,(D94,D95))))),(((D186,D187),D138),(D85,(D156,D157))))),(((((D4,(D9,(((D47,D48),D32),((D160,(D169,D170)),((D105,D106),D100))))),(((D91,D92),D49),(D57,(D64,D65)))),(((((D68,D69),(D196,D197)),D5),((((D107,D108),(D21,D22)),(D11,(D118,D119))),((D140,D141),(((((D161,D162),((D178,D179),(D134,D135))),D31),((D72,D73),(D76,(D171,D172)))),(D75,(D176,D177)))))),((D67,(D152,D153)),(D58,D59)))),((((D30,((D194,D195),D148)),((D50,D51),D39)),((((D163,D164),D56),(((((D111,D112),D97),(D60,(D81,D82))),((D77,D78),D62)),(D34,D35))),D3)),D2)),D0)),((((D139,(D144,D145)),(D79,(D190,D191))),((((D120,D121),((D61,(D142,D143)),D55)),D16),(D7,(D149,(D154,D155))))),((((D63,(D74,(D84,((D93,(D98,D99)),(D180,D181))))),((D103,D104),D52)),(D10,(D12,D13))),((D124,D125),((D165,D166),D133))))),(((D8,((D122,D123),D33)),D1),((D44,(D192,D193)),(((D128,D129),(D132,(D158,D159))),((D184,D185),(((D53,D54),D41),(D28,D29))))))),(((((((E113,E114),(E146,E147)),E16),(E135,E136)),(((E151,E152),(E12,((E44,E45),(((E23,((E33,(((E139,E140),E131),(((E133,E134),E80),E61))),E24)),(E105,E106)),E15)))),((((E53,E54),(E26,((E94,(E160,E161)),E37))),(E88,E89)),E7))),((((E120,E121),E13),((E46,E47),E11)),(((((E111,E112),E28),((E124,E125),E115)),((E141,((E156,E157),E155)),E21)),((E137,(E142,E143)),(E91,(E123,(E196,E197))))))),((((((E74,((E95,((E180,E181),E138)),((E184,E185),E85))),((E65,E66),E60)),(E43,(E96,((E158,E159),(E148,E149))))),((((((E169,E170),E93),(E75,(E177,E178))),(E3,((E52,((E82,E83),(E90,(E92,((E165,E166),E122))))),((((E27,(E31,E32)),E14),(E101,((E190,E191),E179))),((E118,E119),(E86,E87)))))),((((E109,E110),(E63,(E68,E69))),((E153,E154),E67)),E22)),(((((E175,E176),E64),((E167,E168),(E70,E71))),((E144,E145),E55)),(((E116,(E192,E193)),E81),((E84,(E162,E163)),E9))))),((((E78,E79),E30),(((((E57,E58),(E76,E77)),E34),E19),(E1,(((((E20,(((E198,E199),E132),E56)),(((E38,E39),(E62,(((E182,E183),E164),E130))),((E186,E187),E48))),((E51,(E173,E174)),(E104,(E126,E127)))),(E41,E42)),E8)))),(((((E128,E129),(E97,E98)),E59),(((E102,E103),E40),E29)),(((((E188,E189),E117),(E17,E18)),(E150,(E171,E172))),((E5,E6),((E4,(E194,E195)),E0)))))),(((E10,(E49,E50)),E2),(((E35,E36),(((E99,E100),(E107,E108)),(E72,E73))),E25)))))));
