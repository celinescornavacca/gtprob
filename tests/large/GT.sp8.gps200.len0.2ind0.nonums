((((((C,(C,(C,C))),((C,(C,C)),C)),((((C,C),C),((C,C),((C,(C,C)),C))),((((((((C,C),(C,C)),(C,C)),(C,C)),((C,((C,C),C)),((((((C,(C,C)),(C,C)),(C,(C,C))),C),((C,C),(C,(C,C)))),((((((C,C),C),C),(C,C)),(C,C)),(C,C))))),(C,(C,(((C,C),(C,C)),(((C,C),(((C,C),(C,((C,C),C))),(C,(C,C)))),((((C,((C,C),C)),C),(C,C)),((C,C),C))))))),(((((((C,C),(C,C)),(((C,C),C),C)),(((C,C),(((C,C),C),(C,C))),((((C,C),C),(C,C)),(C,(C,C))))),((C,C),C)),((C,C),(C,((C,C),(C,C))))),(((((C,C),((C,C),C)),C),C),(C,((C,(C,C)),(C,C)))))),(((C,(C,(C,(((C,C),((C,C),C)),C)))),((C,C),(C,C))),((C,(C,(C,C))),(((((C,C),((C,C),(((C,C),C),C))),((((C,((C,C),(C,C))),(C,((C,C),(C,(C,C))))),C),(((C,C),(((C,C),C),C)),((C,(C,C)),C)))),C),((((((C,C),C),((C,C),C)),((C,C),(C,C))),(C,C)),(((C,(((C,C),((C,C),C)),((C,C),(C,(C,(C,C)))))),C),C)))))))),(((((((G,(G,G)),((G,G),G)),((((((G,G),G),((G,G),G)),(G,G)),((G,G),((G,(((G,(G,G)),G),G)),(G,(G,G))))),(((((G,G),G),(G,G)),((G,(G,G)),((G,G),((((G,(G,G)),G),(G,G)),(G,G))))),((G,G),((G,G),((G,G),G)))))),((G,G),((((G,(G,G)),((G,G),(((G,(((G,G),((G,G),G)),G)),(((G,G),((G,(G,G)),((G,((G,G),G)),((G,G),(((((G,G),G),G),(G,G)),G))))),G)),((((G,G),(G,G)),((G,G),(G,G))),(G,(G,G)))))),((G,G),((G,((G,((G,G),G)),G)),(G,G)))),(((((G,G),G),G),G),(G,(G,((G,G),G))))))),(((G,G),((((((G,G),G),((G,G),((G,G),(G,G)))),((G,(((G,G),G),((G,G),G))),((G,(G,(G,G))),(((G,G),(G,G)),(G,G))))),G),(((G,G),G),(((((G,G),G),G),G),G)))),(((((G,G),((((G,(G,(G,G))),G),(G,G)),(((G,G),(G,G)),(((((G,G),G),G),((G,(G,(G,G))),((G,G),(G,(G,G))))),G)))),((G,G),G)),(((G,G),G),((((G,G),(G,G)),(((G,G),G),G)),(G,G)))),(G,((G,G),(G,G)))))),((((((((H,H),((H,H),(H,((H,H),(H,(H,(((H,H),H),(H,H)))))))),(((H,H),H),(H,(H,H)))),(((H,H),H),(H,(H,H)))),((((H,H),H),H),((H,((H,H),H)),(H,H)))),(((H,(H,H)),(H,((H,H),(((H,H),H),(H,H))))),(H,((H,(H,(H,H))),(H,H))))),((H,((H,H),(H,H))),((((H,(((H,H),H),H)),(H,H)),H),((((H,((H,H),H)),(H,H)),(H,H)),(H,((H,H),H)))))),((H,H),((H,((((((H,(H,(H,H))),((H,(((H,(H,((H,H),H))),((((H,H),H),(H,H)),(H,((H,H),H)))),(H,H))),H)),(H,H)),(((H,(H,((H,(H,H)),H))),(H,(H,H))),H)),((H,H),(H,H))),((H,((((H,(H,H)),((H,H),(((((H,H),H),(((H,H),H),H)),H),((H,(H,H)),(H,H))))),(((((H,H),H),H),(H,H)),(((H,H),H),((H,H),H)))),H)),(((H,H),H),((H,H),(H,(H,(H,(H,H))))))))),((((((H,H),H),(((H,(H,(H,H))),(H,H)),((H,(H,H)),H))),(((H,H),H),((((H,(H,H)),H),H),(H,(H,(H,H)))))),((H,(H,H)),((((H,H),H),H),(H,((H,H),H))))),(H,H)))))),(((F,F),F),(((((F,(((((F,F),(F,((F,(F,F)),(F,F)))),(F,F)),(F,F)),(((F,F),F),(F,F)))),(F,(F,F))),(((F,(F,F)),(F,((F,F),((F,F),F)))),(((F,((((((F,F),F),F),((((((F,F),F),F),F),(((((F,F),F),F),(F,(F,F))),F)),F)),F),(((F,F),(F,(F,F))),F))),F),(((F,F),F),(F,(((F,(F,F)),F),F)))))),(((F,F),F),F)),((F,((((F,F),F),F),((((F,F),(F,F)),F),(F,(F,F))))),((((((((F,F),((F,((F,F),F)),F)),F),((F,F),F)),(((F,F),F),(F,F))),(((F,F),((F,F),F)),(F,((F,F),((F,F),F))))),(((((F,F),F),F),(((((F,(F,F)),F),((F,F),(F,F))),F),F)),((F,F),F))),(((((((F,(F,F)),(F,F)),(F,F)),((((((F,F),(((F,F),F),(F,F))),((F,F),(F,F))),(((F,F),F),F)),(((F,F),F),(F,F))),(((F,(F,(F,(F,(F,F))))),((F,F),F)),F))),(((F,F),(F,F)),(((F,F),((F,F),(F,(F,F)))),F))),((F,F),F)),(((F,F),F),((((F,F),(F,((F,F),F))),F),((F,(F,F)),((F,F),((F,F),F)))))))))))),((((B,B),(B,B)),((((B,((((B,B),B),((B,B),((B,B),B))),(B,B))),((B,B),B)),(B,(B,B))),((((((B,B),B),B),(B,((B,(B,(B,(B,B)))),B))),((B,B),((B,B),((B,B),(B,(B,B)))))),(((((B,(B,B)),(B,B)),(B,B)),B),((((B,B),(B,(B,(B,B)))),B),(B,B)))))),((((B,B),(((B,B),(B,B)),(((B,B),(B,B)),(B,B)))),((((B,B),((B,((B,B),((((B,B),(B,(B,B))),(B,B)),B))),(((B,B),(B,B)),(((B,B),B),(B,(B,B)))))),((B,B),(B,(B,(B,B))))),((B,B),((((B,(B,(B,(B,B)))),((B,(B,B)),(((B,(B,B)),(B,(B,(B,B)))),B))),(B,B)),((((B,(B,B)),B),B),((B,(B,(B,(B,(B,B))))),B)))))),((((B,(B,(B,(B,((B,B),(B,B)))))),((B,((B,(((B,B),B),B)),B)),(B,(B,B)))),(((((B,B),((((B,B),(B,B)),B),(B,B))),B),(((B,B),(B,B)),B)),((B,B),(B,B)))),(((B,(B,(((B,(B,(B,B))),B),(B,B)))),((((B,B),(B,B)),(B,B)),(((B,(B,B)),B),(B,B)))),(((B,(B,B)),(B,((B,B),(B,B)))),(B,(B,B)))))))),((((((((A,(A,A)),(A,(((A,(A,(A,A))),A),A))),(((((A,(A,A)),A),A),(((A,A),A),(A,A))),((A,A),(A,A)))),((A,A),((((A,A),A),((A,((A,A),A)),A)),((((A,A),(A,A)),(A,A)),(A,(A,A)))))),(((((A,A),(((A,A),((A,A),A)),A)),(A,(A,A))),((A,(((A,A),A),A)),(((((A,A),A),((A,A),A)),(A,A)),A))),((A,(((((A,(A,A)),A),(A,A)),(A,A)),((A,(A,(A,A))),A))),((A,(((A,(A,A)),(A,A)),(A,((A,A),A)))),(A,(A,A)))))),(((A,(A,A)),(A,((A,A),(A,A)))),(((A,(((A,A),(A,A)),((A,A),(A,A)))),A),((((((A,(A,A)),((A,(A,A)),(A,(A,((A,(A,(A,A))),(A,A)))))),(A,((A,A),A))),(((A,A),(((A,A),A),(((A,A),A),(A,A)))),A)),(A,A)),(((A,(A,A)),A),(A,A)))))),((((A,A),(A,A)),((A,(A,A)),(A,A))),((A,((A,A),(A,A))),((((((A,A),((((A,A),(A,A)),((A,A),A)),(A,A))),(A,(((A,A),(A,A)),A))),((((A,A),A),(A,A)),(((A,A),A),A))),(((A,A),A),(A,(A,((A,A),(A,A)))))),A)))),(((((((((((((D,D),((((D,(D,D)),D),((D,D),D)),D)),D),((D,D),D)),(D,D)),((D,D),((D,D),D))),((((D,((D,D),D)),D),(D,D)),((D,D),((D,D),(D,D))))),((((D,D),(D,((D,D),D))),((D,D),(((D,D),D),(D,D)))),(D,((D,(D,D)),D)))),(((((D,D),(D,D)),D),((D,D),(D,(D,(D,D))))),(((D,D),D),(D,(D,D))))),(((((D,(D,(((D,D),D),((D,(D,D)),((D,D),D))))),(((D,D),D),(D,(D,D)))),(((((D,D),(D,D)),D),((((D,D),(D,D)),(D,(D,D))),((D,D),(((((D,D),((D,D),(D,D))),D),((D,D),(D,(D,D)))),(D,(D,D)))))),((D,(D,D)),(D,D)))),((((D,((D,D),D)),((D,D),D)),((((D,D),D),(((((D,D),D),(D,(D,D))),((D,D),D)),(D,D))),D)),D)),D)),((((D,(D,D)),(D,(D,D))),((((D,D),((D,(D,D)),D)),D),(D,(D,(D,D))))),((((D,(D,(D,((D,(D,D)),(D,D))))),((D,D),D)),(D,(D,D))),((D,D),((D,D),D))))),(((D,((D,D),D)),D),((D,(D,D)),(((D,D),(D,(D,D))),((D,D),(((D,D),D),(D,D))))))),(((((((E,E),(E,E)),E),(E,E)),(((E,E),(E,((E,E),(((E,((E,(((E,E),E),(((E,E),E),E))),E)),(E,E)),E)))),((((E,E),(E,((E,(E,E)),E))),(E,E)),E))),((((E,E),E),((E,E),E)),(((((E,E),E),((E,E),E)),((E,((E,E),E)),E)),((E,(E,E)),(E,(E,(E,E))))))),((((((E,((E,((E,E),E)),((E,E),E))),((E,E),E)),(E,(E,((E,E),(E,E))))),((((((E,E),E),(E,(E,E))),(E,((E,((E,E),(E,(E,((E,E),E))))),((((E,(E,E)),E),(E,((E,E),E))),((E,E),(E,E)))))),((((E,E),(E,(E,E))),((E,E),E)),E)),(((((E,E),E),((E,E),(E,E))),((E,E),E)),(((E,(E,E)),E),((E,(E,E)),E))))),((((E,E),E),(((((E,E),(E,E)),E),E),(E,(((((E,(((E,E),E),E)),(((E,E),(E,(((E,E),E),E))),((E,E),E))),((E,(E,E)),(E,(E,E)))),(E,E)),E)))),(((((E,E),(E,E)),E),(((E,E),E),E)),(((((E,E),E),(E,E)),(E,(E,E))),((E,E),((E,(E,E)),E)))))),(((E,(E,E)),E),(((E,E),(((E,E),(E,E)),(E,E))),E)))))));
