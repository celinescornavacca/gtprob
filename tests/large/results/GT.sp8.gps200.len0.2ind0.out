((((((C86,(C97,